pkg/term/term_windows.go

```diff
@@ -1,12 +1,15 @@
 // +build windows
+
 package term
 
 import (
+	"fmt"
 	"io"
 	"os"
+	"os/signal"
 
-	"github.com/Sirupsen/logrus"
-	"github.com/docker/docker/pkg/term/winconsole"
+	// "github.com/Sirupsen/logrus"
```

Comment:
What I meant was the fact that you're printing all (newly implemented?) error/debug/etc. messages via `fmt` instead of via `logrus`